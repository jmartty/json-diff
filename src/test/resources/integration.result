JsonPrimitive diff, expected: 89 but was 88
Stack[3]: Array[1]
Stack[2]: Object["values"]
Stack[1]: Array[2]
Stack[0]: Object["somekey"]