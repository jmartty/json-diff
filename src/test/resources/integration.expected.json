{
  "somekey": [
    1,
    2,
    {
      "another": true,
      "values": [
        "chau",
        89
      ]
    }
  ]
}